training/Deepspeed-Domino/domino/arguments.py

```diff
@@ -0,0 +1,366 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
I observe three issues with this file during testing
1. It is too different from the original megatron [version](https://github.com/NVIDIA/Megatron-LM/blob/23.08/megatron/arguments.py). This seems to because other codes were added. I understand the convenience of fewer files. However, in this case it is better to keep things as similar to original megatron as much as possible. We want the diff to be as minimal so that developers can easily understand the difference. Also, it will make it easier for Nvidia to consider upstreaming Domino into Megatron-LM, which will be a good thing. 
2. `--no-gradient-accumulation-fusion` is missing in the argument handling even though it is in original [megatron](https://github.com/NVIDIA/Megatron-LM/blob/375395c187ff64b8d56a1cd40572bc779864b1bd/megatron/arguments.py#L848C24-L848C59) and used in [pretrain_gpt.sh](https://github.com/microsoft/DeepSpeedExamples/blob/addf1f1a7cb6d8c7f041e2ab724e5964384c3912/training/Deepspeed-Domino/pretrain_gpt.sh#L49). Is this option supported by Domino?
3. `--local_rank` has changed to `--local-rank`, which is a difference to [megatron](https://github.com/NVIDIA/Megatron-LM/blob/375395c187ff64b8d56a1cd40572bc779864b1bd/megatron/arguments.py#L1037) and also breaks `deepspeed` launcher. Can you please explain the rationale for this change?